packages/apps/reputation-oracle/server/src/modules/auth/auth.service.ts

```diff
@@ -83,13 +111,18 @@ export class AuthService {
     return this.auth(userEntity);
   }
 
-  public async signup(data: UserCreateDto): Promise<UserEntity> {
+  public async signup(data: UserCreateDto, ip?: string): Promise<UserEntity> {
```

Comment:
add hcaptcha token verification even though you leave it commented

Reply:
added and commented